Looms.PoS/Looms.PoS.Application/Features/Product/Commands/DeleteProduct/DeleteProductCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Looms.PoS.Application.Abstracts;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Looms.PoS.Application.Features.Product.Commands.DeleteProduct;
+
+public record DeleteProductCommand : LoomsHttpRequest, IRequest<IActionResult>
+{
+    public string Id { get; init; }
```

Comment:
empty line after properties

Reply:
fixed in "fixed indentation/layout problems, changed Variation to Variation request and made create and update requests take IEnumerable<VariationRequest>"